### ISSUE TITLE
Unable to install app

### ISSUE THREAD

**Issue by joshfermin** [FUSAKLA/gdocs-songbook#11]
https://workspace.google.com/marketplace/app/songbook/591739870688

When attempting to install this App I am getting the following error:
<img width="653" alt="Screenshot 2024-11-26 at 4 12 32 PM" src="https://github.com/user-attachments/assets/395ad839-16bb-44b2-a34b-4b98e045ad62">

**Comment by FUSAKLA** [comment]
Oh my.. sorry to hear that.

I'm afraid this might be a bigger issue 😖 

I'll try to look at it asap

**Comment by jpegazo** [comment]
Tengo el mismo problema :(

Ayuda!

**Comment by FUSAKLA** [comment]
Hi, sorry for the long inactivity.

I have sad news, since last year Google started to require a [security assessment(CASA)](https://developers.google.com/identity/protocols/oauth2/production-readiness/restricted-scope-verification) for all apps, that has access to all your Google Drive files (which this plugin uses for importing songs to new files, creating table of contents or printing songbooks) 

As I looked, this would mean thousands of dollars worth audit annually, which is way out of scope for this free time project :/

I'm thinking about creating some "stripped" version without the functionality that requires such high security check, but I struggle to find the time and motivation since I really liked the TOC feature for example :/

**Comment by Broseten** [comment]
Yeah. Same issue here....that's unfortunate. Anyway, thanks for the great work! <3

I figured since it is open-source I might be able to deploy it myself and then there should be no issue with this "low security" app right? Which might be trickier for non-technical users, but otherwise it would hopefully work the same.

**Comment by FUSAKLA** [comment]
@Broseten that is indeed an option and should work. 

If you manage to pull that off on yourself, it would be great if you could paste some HOWTO for others to try it 🙏 

Otherwise, I might try to write it up myself at some point, hopefully.

**Comment by dhananjeyan12** [comment]
Had the same issue when I tried installing. If the google security CASA is the problem, can you just remove the import song feature and release so that we can use the other features like the chords, transposing the chords and everything. It'll be a great help if you do this. Thank you

**Comment by FUSAKLA** [comment]
Actually importing of the songs isn't the issue, the issue is the TOC feature and printing IIRC because that requires access to your drive which is potentially dangerous.

Maybe the TOC feature could be done without need for the drive API if one would generate it inside the current file 🤔 

The thing is, I'm a bit short of time lately, but will try to find awhile.

**Comment by fertaricco** [comment]
Hola! Me gustaría mucho poder instalarlo. Avisame si puedo echar una mano con algo, una guía para saber dónde tocar y que podamos volver a usar el Complemento.
